loops-and-arrays.js

```diff
@@ -12,6 +12,11 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
+for (i=0; i<names.length; i++) {
```

Comment:
De i kun je beter declareren met een let keyword, de idea geeft bij mij een kringeltje ter indicatie dat je hier iets mee moet.
Als je er let i = 0; van maakt zie je dat ie weer dik tevreden is. 
Met control+alt+l kun je de opmaak automatisch verbeteren, selecteer de code alvorens de toetscombinatie te gebruiken: er komen dan spaties om de operatoren. Lekker makkelijk.